### PR TITLE
[SDAO-95] macOS brew setup script

### PR DESCRIPTION
**Pull request by gpevnev**
[//]: # (This is a template of a pull request template.)
[//]: # (You should modify it considering specifics of a particular repository and put it there.)
[//]: # (Comments like this are meta-comments, they shouldn't be present in the final template.)
[//]: # (Comments starting with '

[//]: # (Here you can add a link to the corresponding issue tracker, e. g. https://issues.serokell.io/issue/SDAO-)
## Related issue(s)
https://issues.serokell.io/issue/SDAO-95

#### Related changes (conditional)

- Tests
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from
        silently reappearing again.

[//]: # (Add more docs here if you have them in the repository)
- Documentation
  - [ ] I checked whether I should update the docs and did so if necessary:
    - [README](../tree/master/README.md)
    - Haddock

[//]: # (Mostly for public repositories)
[//]: # (Recording changes is optional, depends on repository, useful for some libs)
- Public contracts
  - [ ] Any modifications of public contracts comply with the [Evolution
  of Public Contracts](https://www.notion.so/serokell/Evolution-of-Public-Contracts-2a3bf7971abe4806a24f63c84e7076c5) policy.
  - [ ] I added an entry to the [changelog](../tree/master/CHANGES.md) if my changes are visible to the users
        and
  - [ ] provided a migration guide for breaking changes if possible

#### Stylistic guide (mandatory)

[//]: # (Update link to style guide if necesary or remove if it's not present)

- [ ] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).
- [ ] My code complies with the [style guide](../tree/master/docs/code-style.md).

### CODE REVIEW

File: scripts/macos-install.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#! /usr/bin/env bash
+tezos_client_version="2020-03-11-ledger-patched"
+
+brew tap gpevnev/stakerdao
```
Comment (rvem) [inline scripts/macos-install.sh]: you should probably to ask ops to create "Serokell" account for brew stuff

File: scripts/macos-install.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+mkdir -p config_dir
+
+rm $config_file
+
+touch $config_file
+# TODO: figure out why writing to config file does not work
+echo "envTezosClientCmd: \"$tezos_client\"" > $config_file
+echo 'envNodeAddr:' >> $config_file
+echo '    naHost: "carthage.testnet.tezos.serokell.team"' >> $config_file
+echo '    naPort: 8732' >> $config_file
```
Comment (gpevnev) [inline scripts/macos-install.sh]: Also this does not work somehow, and I dunno why. @rvem @georgeee any ideas?

**Comment by rvem** [comment]
also, please run `shellcheck` for your scripts :upside_down_face:

**Comment by gpevnev** [comment]
@rvem what is that?

**Comment by rvem** [comment]
https://www.shellcheck.net/

**Pull request closed without merging**